nc2zarr/opener.py

```diff
@@ -24,6 +24,9 @@
 import warnings
 from typing import List, Optional, Iterator, Callable, Union, Dict, Hashable
 
+import s3fs
+s3 = s3fs.S3FileSystem(anon=False)
+
```

Comment:
1. Having `s3` as global variable is not acceptable.
2. `anon=True` is an assumption that is wrong is almost all cases. Your use case is an exception.

The filesystem used for the inputs must be made configurable in the configuration file.